README.md

```diff
@@ -76,4 +76,4 @@ With Activity, modeling business processes turns out to be ridiculously simple:
 
 ## Operation
 
-Trailblazer's [`Operation`](http://trailblazer.to/2.1/operation) internally uses an activity to model the processes.
+Trailblazer's [`Operation`](http://trailblazer.to/gems/operation/2.0/) internally uses an activity to model the processes.
```

Comment:
This reference to version 2.0, while the current version is 2.1

Reply:
hmm, there is no http://trailblazer.to/gems/operation/2.1/ so I link to available `2.0`: http://trailblazer.to/gems/operation/2.0/ 😅